spec/system/candidate_interface/references/candidate_sends_a_reminder_email_spec.rb

```diff
@@ -1,69 +0,0 @@
-require 'rails_helper'
```

Comment:
check if this is replicated

Reply:
This (and the reference history one) are replicated here:
spec/system/candidate_interface/references/candidate_views_their_references_on_the_post_offer_dashboard_spec.rb